dlp/snippets/README.md

```diff
@@ -0,0 +1,7 @@
+# Google Data Loss Prevention
+
+Docs: https://cloud.google.com/dlp/docs/
+
+`Google Data Loss Prevention` provides programmatic access to a powerful
```

Comment:
let me check on this real quick ... this summary has changed dramatically in production to

Cloud DLP enables enterprises to automatically discover, classify, and protect their most sensitive data elements.

Running it by one other person. I'll get right back to you.

Reply:
Got it. 

branding must either be:

Long name: Cloud Data Loss Prevention
Short name: Cloud DLP

So change this to 

Cloud Data Loss Prevention enables enterprises to automatically discover, classify, and protect their most sensitive data elements.